superbench/benchmarks/micro_benchmarks/ort_inference_performance.py

```diff
@@ -0,0 +1,187 @@
+
+    def add_parser_arguments(self):
+        """Add the specified arguments."""
+        super().add_parser_arguments()
+
+        self._parser.add_argument(
+            '--pytorch_models',
+            type=str,
+            nargs='+',
+            default=self._pytorch_models,
+            help='ONNX models for TensorRT inference benchmark, e.g., {}.'.format(', '.join(self._pytorch_models)),
+        )
+
+        self._parser.add_argument(
+            '--precision',
+            type=Precision,
+            choices=[Precision.FLOAT32, Precision.FLOAT16, Precision.INT8],
+            default=Precision.FLOAT16,
+            required=False,
+            help='Precision for inference, allow int8, float16, or float32 only.',
+        )
+
+        self._parser.add_argument(
+            '--graph_opt_level',
+            type=int,
+            default=3,
+            choices=[0, 1, 2, 3],
+            required=False,
+            help='ONNXRuntime graph optimization level, 0 for ORT_DISABLE_ALL, 1 for ORT_ENABLE_BASIC, '
+            '2 for ORT_ENABLE_EXTENDED, 3 for ORT_ENABLE_ALL.',
+        )
+
+        self._parser.add_argument(
+            '--batch_size',
+            type=int,
+            default=32,
+            required=False,
+            help='Set batch size for inference.',
+        )
+
+        self._parser.add_argument(
+            '--num_warmup',
+            type=int,
+            default=64,
+            required=False,
+            help='The number of warmup step before the benchmarking.',
+        )
+
+        self._parser.add_argument(
+            '--num_steps',
+            type=int,
+            default=256,
+            required=False,
+            help='The number of test step for benchmarking.',
+        )
+
+    def _preprocess(self):
+        """Preprocess/preparation operations before the benchmarking.
+
+        Return:
+            True if _preprocess() succeed.
+        """
+        if not super()._preprocess():
+            return False
+
+        for model in self._args.pytorch_models:
+            if hasattr(torchvision.models, model):
+                data_type = Precision.FLOAT16.value if self._args.precision == Precision.FLOAT16 \
+                    else Precision.FLOAT32.value
+                model_path = f'{self.__model_cache_path / (model + "." + data_type + ".onnx")}'
+                torch.onnx.export(
+                    getattr(torchvision.models, model)(pretrained=True).to(dtype=getattr(torch, data_type)).cuda(),
+                    torch.randn(self._args.batch_size, 3, 224, 224, device='cuda', dtype=getattr(torch, data_type)),
+                    model_path,
+                    input_names=['input'],
+                )
+                if self._args.precision == Precision.INT8:
+                    file_name = '{model}.{precision}.onnx'.format(model=model, precision=self._args.precision)
+                    quantize_dynamic(model_path, f'{self.__model_cache_path / file_name}')
```

Comment:
Besides, based on the document,
```
In general, it is recommended to use dynamic quantization for RNN and transformer-based models, and static quantization for CNN models.
```

Reply:
As the doc introduced,
> The main difference between dynamic quantization and static quantization is how scale and zero point of activation is calculated. For static quantization, they are calculated offline with calibration data set. All the activations have same scale and zero point. While for dynamic quantization, they are calculated on flight and will be specific for each activation, thus they are more accurate but introduce extra computation overhead.

So static quantization can be regarded as an improvement feature, which improved the performance but lost some accuracy compared with dynamic quantization. For CNN models, the memory bandwidth for activations is more heavy than transformer-based models, so the docs suggest to use static quantization for CNN models.
I suggest we use the dynamic quantization which is a more common way. For static quantization, we have to prepare one calibration dataset for offline calculation.